packages/app/lib/nats.ts

```diff
@@ -33,6 +33,14 @@ function connectToNats() {
 
             stan.on('error', (error: any) => {
                 log.error(error) //? do anything beyond logging?
+
+                if (stan) {
+                    log.info('Stan is being resolved...', JSON.stringify(stan))
```

Comment:
Not sure this log needs to be seen in production, can we switch this to log.debug so we can keep the logs clean?

Reply:
Sure, we can change that to debug.

Reply:
Resolved in [36b4556](https://github.com/nasa/gesdisc-meditor/pull/47/commits/36b45564a2a0b939f94777bbfdec0ac7ee4e62a7).